include/iarc7_motion/ThrustModel.hpp

```diff
@@ -142,6 +157,7 @@ struct ThrustModel
 
         if(zero_voltage_thrust >= desired_thrust) {
             start_thrust = desired_thrust;
+            //ROS_ERROR("ZERO VOLTAGE THRUST");
```

Comment:
is this supposed to be commented out or even here?

Reply:
Yes